sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawVip.java

```diff
@@ -24,4 +30,12 @@ public int getPort() {
     public String getPrefix() {
         return prefix;
     }
+
+    public String getProtocol() {
+        return protocol;
+    }
+
+    public Boolean isAdvertise() {
```

Comment:
Does the serialization logic require this name? If not, maybe `shouldAdvertise` or `isAdvertised`?